client/src/ui/NotFound.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+export default function NotFound() {
+  return (
+    <div className="col-md-12 col-sm-12 col-xs-12 not-found-padding-top">
+      <img src="/noResults.svg" className="img-responsive no-results" alt="No class found" ></img>
```

Comment:
```suggestion
      <img src="/noResults.svg" className="img-fluid no-results" alt="No class found" ></img>
```
see [Bootstrap 4 migration guide](https://getbootstrap.com/docs/4.0/migration/#images)

Reply:
Fixed!